### ISSUE TITLE
The accuracy is low when i train on my own dataset

### ISSUE THREAD

**Issue by jzx-gooner** [kwotsin/TensorFlow-ENet#30]
I want to use this model for my own dataset(classes 2:car and backgroud)
However the accuracy is so low 
with the loss falling the mean_iou is falling 。the model is not work for this dataset  
what is the problem？

**Comment by Ariel-JUAN** [comment]
Hi, have you solve the problem？ I met this problem, too.

**Comment by c13proto** [comment]
hi, me too. black image created. 
do you know solution or other fast segmentation network for 2 class ?
